client/run.js

```diff
@@ -3,13 +3,14 @@ var admin = require('./')
   , api = require('./api')
 
 // for debugging
-window.React = require('react')
+window.React = require('react');
 
-api.init('rest', '/admin/api');
+var url = window.location.href.replace(/^.*\/\/[^\/]+/, '').split('/');
+var rootPath = url.slice(0, url.indexOf('admin')).join('/');
```

Comment:
this won't work if my rootPath is something like `/my/admin/site/`, but it's plenty good for now